app/views/subject_rule_effects/_form.html.erb

```diff
@@ -1,17 +1,23 @@
 <%= simple_form_for(@subject_rule_effect, url: [@workflow, @subject_rule, @subject_rule_effect]) do |f| %>
   <%= f.error_notification %>
 
-  <div class="form-inputs">
+  <div class="form-inputs form">
```

Comment:
Maybe swap the two around, so that the general `form` goes before the specfic `form-inputs`? Very minor tho.